src/Backend/Modules/Location/Actions/Add.php

```diff
@@ -74,6 +74,17 @@ private function validateForm(): void
                 // insert the item
                 $item['id'] = BackendLocationModel::insert($item);
 
+                $generalSettings = $this->get('fork.settings')->getForModule('Location');
```

Comment:
You need to write extra code that when these settings change you update all locations that don't overwrite the settings

Reply:
added update script on settings page 03dae81bd594410725572bee8e1aad4459176665